src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/install-android-apps-gradle-android-studio.mdx

```diff
@@ -46,63 +46,59 @@ If you haven't already, create your free New Relic account below to start monito
 As part of the installation process for mobile monitoring, New Relic automatically generates an [application token](/docs/mobile-apps/viewing-your-application-token). This is a 40-character hexadecimal string for authenticating each mobile app you monitor in New Relic.
 
 1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile**.
-2. If applicable: From the [**Mobile apps** index](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-index), select **Add a new app**.
+2. If applicable: From the [**Mobile apps** index](/docs/mobile-monitoring/mobile-monitoring-ui/mobile-app-pages/mobile-apps-index), select <Icon name="fe-plus-circle"/>
 3. From the **Get started** page, select **Android** as the platform for mobile monitoring.
-4. Type a meaningful name for your mobile app, and select **Continue**.
+4. Enter a name for your mobile app, and select **Continue**.
 
-Continue with the steps to [configure mobile monitoring](#configuration) with Gradle and Android Studio.
-
-OR:
-
-* To complete the configuration process for a new mobile app later: Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile**, then select **See instructions** next to your mobile app name.
-* To upgrade an existing Android installation: Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Settings > Installation**.
+Continue with the steps to [configure mobile monitoring](#configuration) with Gradle and Android Studio
 
 ## Configure with Gradle and Android Studio [#configuration]
 
 These procedures to configure your Android app with Gradle and Android Studio also appear on the **Get started** page in New Relic.
```

Comment:
Same comment as above, consider adding context as to where I need to follow these steps. Example:

"After installing our Android agent, you'll need to make some configurations to your Android app to successfully report data to New Relic. Follow the steps below in your app's Android Studio."